Examples/Sample_xDscWebServiceRegistrationWithSecurityBestPractices.ps1

```diff
@@ -53,7 +53,7 @@ configuration Sample_xDscWebServiceRegistrationWithEnhancedSecurity
             DependsOn               = "[WindowsFeature]DSCServiceFeature" 
             RegistrationKeyPath     = "$env:PROGRAMFILES\WindowsPowerShell\DscService"   
             AcceptSelfSignedCertificates = $true
-            UseUpToDateSecuritySettings = $true
+            UseSecurityBestPractices = $true
```

Comment:
We are not only adding one entry. We are changing the entries under SCHANNEL\Protocols. Before applying the configuration, user may have other settings of protocols. That is, for all the protocols we listed at top of SecureTLSProtocols.psm1, user may have a different set of on/off for each of them. What I mean about "not knowing the previous setting" is that we won't know what user turned on and off before apply BestPractice configuration.